packages/frontend/src/app/home/Home.js

```diff
@@ -47,52 +48,69 @@ function Home () {
-      Api.getBlocks(1, 1, 'desc')
-    ])
-      .then(([
-        status,
-        paginatedTransactions,
-        paginatedDataContracts,
-        paginatedRichestIdentities,
-        paginatedTrendingIdentities,
-        transactionsHistory,
-        latestBlocks
-      ]) => {
-        const [latestBlock] = latestBlocks.value?.resultSet ? latestBlocks.value.resultSet : {}
-        setStatus({
-          latestBlock,
-          ...status.value
-        })
-        setDataContracts(paginatedDataContracts.value.resultSet)
-        setTransactionsHistory(transactionsHistory.value ? convertTxsForChart(transactionsHistory.value) : [])
-        setTransactions(state => ({
-          ...state,
-          items: paginatedTransactions.value?.resultSet
-            ? paginatedTransactions.value.resultSet
-            : []
-        }))
-        setRichestIdentities(state => ({
+      Api.getStatus().then(res => {
+        setStatus(state => ({
           ...state,
-          items: paginatedRichestIdentities.value?.resultSet
-            ? paginatedRichestIdentities.value.resultSet
-            : []
+          ...res,
+          loaded: true
         }))
-        setTrendingIdentities(state => ({
+      }),
+      Api.getTransactions(1, 10, 'desc')
+        .then(paginatedTransactions => {
+          setTransactions(state => ({
+            ...state,
+            items: paginatedTransactions?.resultSet
+              ? paginatedTransactions.resultSet
+              : [],
+            loaded: true
+          }))
+        }),
+      Api.getDataContracts(1, dataContracts.printCount, 'desc', 'documents_count')
+        .then(paginatedDataContracts => setDataContracts(state => ({
           ...state,
-          items: paginatedTrendingIdentities.value?.resultSet
-            ? paginatedTrendingIdentities.value.resultSet
-            : []
-        }))
-      })
+          items: paginatedDataContracts?.resultSet || [],
+          loaded: true
+        }))),
+      Api.getIdentities(1, 10, 'desc', 'balance')
+        .then(paginatedRichestIdentities => {
+          setRichestIdentities(state => ({
+            ...state,
+            items: paginatedRichestIdentities?.resultSet
+              ? paginatedRichestIdentities.resultSet
+              : [],
+            loaded: true
+          }))
+        }),
+      Api.getIdentities(1, 10, 'desc', 'tx_count')
+        .then(paginatedTrendingIdentities => {
+          setTrendingIdentities(state => ({
+            ...state,
+            items: paginatedTrendingIdentities?.resultSet
+              ? paginatedTrendingIdentities.resultSet
+              : [],
+            loaded: true
+          }))
+        }),
+      Api.getTransactionsHistory(transactionsChartConfig.timespan.default)
+        .then(transactionsHistory => {
+          setTransactionsHistory(state => ({
+            ...state,
+            data: transactionsHistory ? convertTxsForChart(transactionsHistory) : [],
```

Comment:
I think it will be better if we map values at the view level. Since transactionsHistory is used in one place, we don't need to convert it in two places